Misc/NEWS.d/next/Security/2024-06-25-04-42-43.gh-issue-112301.god4IC.rst

```diff
@@ -0,0 +1 @@
+Add default compiler options to improve security
```

Comment:
Please list concrete compiler options you added.

Reply:
Added options to news file